framework/generated/vulkan_generators/vulkan_cpp_consumer_body_generator.py

```diff
@@ -795,8 +795,6 @@ def makeConsumerFuncBody(self, returnType, name, values):
             if arg.base_type in self.handle_names:
                 handleObjectType = makeObjectType(arg.base_type)
 
-            body += makeGen('// {arg.name}', locals(), indent=0)
```

Comment:
I'm ok to remove this code to remove some debug output. But the code was added by @MarkY-LunarG. I wonder if @MarkY-LunarG agree on it?

Reply:
I think removing it sounds right.  @MarkY-LunarG

Reply:
I'm okay with removing this.